src/Core/Prefab/GlobeView.js

```diff
@@ -83,6 +83,7 @@ class GlobeView extends View {
      * @param {CameraTransformOptions|Extent} placement - An object to place view
      * @param {object} [options] - See options of {@link View}.
      * @param {Object} [options.controls] - See options of {@link GlobeControls}
+     * @param {Boolean} [options.activateSunlight=true] - whether to add a THREE.DirectionalLight simulating sun lightning
```

Comment:
I know this is a breaking change but was there a reason to put a `DirectionalLight` in an arbitrary position?
I'm also for renaming `activateSunlight` to `sunlight` or pass a `light` parameter of type `Light` and defaulting to the current sun light.